### ISSUE TITLE
Public Comment 1.6 Editorial and improve English

### ISSUE THREAD

**Issue by chris-little** [opengeospatial/Temporal-Abstract-Spec#62]
Public Comment received  2023-12-11
**Abstract** 
Missing blank in "...terminology.The..."

**Preface**
 "... Because of this effort, many people use calendar based "coordinates", with the attendant ambiguities, imprecision and inappropriate scope." Substantiate the claim indicated by "Because of this effort". Is it really "because"?

In general, wording is too vague, the term "attendant ambiguities", while being rhythmically attractive, leaves me baffled.

"people" Inappropriate titulation. End users?

**Normative references**
There are some problems with closing paranthesis and some unmotivated ellipsis in the two references to ISO documents.

**Terms, definitions...**
Datum + Reference frame: The indicator "Admitted" is duplicated in two different typographical designs.

The definition given for the term "reference frame" (and reference system) is geodetically dubious, but that will be a problem to tackle in connection with the next ISO-19111 and/or ISO-19161 revision.

**Temporal regimes**
"To more clearly think about time, this Abstract Specification adopts the term "Regime" to..." Inappropriate anthropomorpization: The specification is not a person. It cannot think and it cannot adopt. Suggestion: "To enable more clear reasoning about time, ... we adopt, for the purposes of this Abstract Specification, ..."

"Paradoxically, the calendar regime has historically driven advances in mathematics and physics." Citation needed.

**Calendars**
"Allen Operators" - The link is invisible.

"Calendrical Calculations" - The link is invisible. And perhaps the typographical representation should make it more clear that this is a reference to a book, rather than a sentence starting with the words "calendrical calculations" (or perhaps just use the numerical reference "[5]").

"There is no simple arithmetic." - This may be correct English, but to this non-native speaker, it sounds like a half-sentence is missing. I suggest "There is typically no simple arithmetic(al?) connection between calendar systems and continuous timescales".

**Other regimes**
"Astronomical Algorithms" - same problem as for "Calendrical Calculations" above: Not sufficiently clear that this is a reference.

Since the speed of light, c , in a vacuum is a measurable constant:
a. remove the blank after c;
b. Given that the numerical value of c is *by definition* 299_792_458 m/s, it can hardly be measured. It can be observed, though. So I recommend the wording "observable constant".

"space weather approaching from the sun" I'm unsure, but I believe we're first talking about space weather once it creates disturbances in the ionosphere. So probably rather something like "the prediction of space weather events, due to eruptions approaching from the sun".

**Accountancy**
"The financial and administrative domains often use weeks, quarters, and other calendrical measures. These may be convenient (though often not!) for the requisite tasks, but are usually inappropriate for scientific or technical purposes."

- I believe the "may" in "may be convenient" provides the same meaning as the parenthetical "though often not", which is, in that case pleonastic and should be left out.

- The "usually" in "usually inappropriate" covers the case of GPS weeks. This exceptional case may, however, deserve (footnote?) mentioning, to proactively shoot down irrelevant commentary from the GNSS community?

**Events**
Avoid the cultural stereotypes "Xi" and "Yi". Perhaps replace by e.g. (NN, MM), (A, B) or (X, Y).

"though not completely" -> though often not completely / though seldom completely / though typically not completely.

**Temporal CRS**
I repeat my remark that the term CRS is geodetically dubious, but also that this is not to be handled here, but in future revisions of ISO-19111/19161/19127.

**Calendar RS**
"Calendar...Calculated algorithms": I believe algorithms are *derived* (not calculated), then used for calculations.

**Clock**
Typo in "4.4m": missing "half space" ("\," in TeX lingo) between number and unit.

**Examples**
Example 1 needs too many "ifs and buts" (about stable climate, stable icesheets, and about the mechanics of snow compaction via firn into ice), to be truly useful. I suggest this wording:

"Example 1: A long, deep ice core is retrieved from an ice sheet. From chemical identification of layers representing known large scale volcanic eruptions, the connection between depth and time is known, so length can be converted to time. This enables the dates of some previously unknown large scale volcanic eruptions to be identified and timed."

**Synchronization**
"However, if the clocks are moving with respect to each other, they cannot be precisely coordinated (unless the communication is instantaneous)." The parenthetical reservation is unphysical, and should be left out.

 Suggestion: "However, if the clocks are moving with respect to each other, perfect synchronization would require communication to be instantaneous." (I realize that the original text said "coordinated", not "synchronized", and I appreciate the difference. Feel free to use "perfect coordination", if that is a better fit).

Also: another case of badly delineated references (to "A brief history of timekeeping").

**Annex A Conformance tests**
Missing!

**Annex B Glossary**
It is unclear what entities belong here, vs. chapter 4 (Terms, definitions and abbreviated terms) Also, the problem with the typesetting of "ADMITTED" reoccurs here.

**Comment by chris-little** [comment]
Responses to comments received  2023-12-11
**Abstract** 
Missing blank in "...terminology.The..." ACCEPTED.

**Preface**
 "... Because of this effort, many people use calendar based "coordinates", with the attendant ambiguities, imprecision and inappropriate scope." ACCEPTED. Replaced "Because..." by "A consequence ..."

 "attendant ambiguities", ACCEPTED. Replaced by "associated ambiguities in underlying algorithms, ..."
 
"people" ACCEPTED. Replaced by "end-users and software developers"

**Normative references**
There are some problems with closing paranthesis and some unmotivated ellipsis in the two references to ISO documents

**Terms, definitions...**
Datum + Reference frame: The indicator "Admitted" ACCEPTED. Metanorma error in production system.

The definition given for the term "reference frame" (and reference system) is geodetically dubious, but that will be a problem to tackle in connection with the next ISO-19111 and/or ISO-19161 revision. NOTED

**Temporal regimes**
"To more clearly think about time, this Abstract Specification adopts the term "Regime" to..." Inappropriate anthropomorpization: The specification is not a person. It cannot think and it cannot adopt. Suggestion: "To enable more clear reasoning about time, ... we adopt, for the purposes of this Abstract Specification, ..."

"Paradoxically, the calendar regime has historically driven advances in mathematics and physics." Citation needed.

**Calendars**
"Allen Operators" - The link is invisible. ACCEPTED. Link visible in GitHub. Metanorma error in production system.

"Calendrical Calculations" - The link is invisible. ACCEPTED. Link visible in GitHub. Metanorma error in production system.

"There is no simple arithmetic." ACCEPTED. Replaced by "Typically there is no simple arithmetical connection between calendar systems and timescales". This is to avoid confusion between `continuous` and `discrete` timescales, both of which are not discontinuous.

**Other regimes**
"Astronomical Algorithms" - same problem as for "Calendrical Calculations" above:  ACCEPTED. Link visible in GitHub. Metanorma error in production system. Not sufficiently clear that this is a reference.

Since the speed of light, c , in a vacuum is a measurable constant:
a. remove the blank after c
b. Given that the numerical value of c is *by definition* 299_792_458 m/s, it can hardly be measured. It can be observed, though. So I recommend the wording "observable constant"

"space weather approaching from the sun" I'm unsure, but I believe we're first talking about space weather once it creates disturbances in the ionosphere. So probably rather something like "the prediction of space weather events, due to eruptions approaching from the sun"

**Accountancy**
"The financial and administrative domains often use weeks, quarters, and other calendrical measures. These may be convenient
(though often not!) for the requisite tasks, but are usually inappropriate for scientific or technical purposes."

- I believe the "may" in "may be convenient" provides the same meaning as the parenthetical "though often not", which is, in that case
pleonastic and should be left out.

- The "usually" in "usually inappropriate" covers the case of GPS weeks. This exceptional case may, however, deserve (footnote?)
mentioning, to proactively shoot down irrelevant commentary from the GNSS community?

**Events**
Avoid the cultural stereotypes "Xi" and "Yi". Perhaps replace by e.g. (NN, MM), (A, B) or (X, Y)

"though not completely" -> though often not completely / though seldom completely / though typically not completely

**Temporal CRS**
I repeat my remark that the term CRS is geodetically dubious, but also that this is not to be handled here, but in future revisions of ISO-19111/19161/19127

**Calendar RS**
"Calendar...Calculated algorithms": ACCEPTED. Replaced by "derived algorithms"

**Clock**
Typo in "4.4m": missing "half space" ("\," in TeX lingo) between number and unit. ACCEPTED.

**Examples**
Example 1 needs too many "ifs and buts" (about stable climate, stable icesheets, and about the mechanics of snow compaction via firn into ice), to be truly useful. I suggest this wording:

"Example 1: A long, deep ice core is retrieved from an ice sheet. From chemical identification of layers representing known large scale volcanic eruptions, the connection between depth and time is known, so length can be converted to time. This enables the dates of some previously unknown large scale volcanic eruptions to be identified and timed." ACCEPTED.

**Synchronization**
"However, if the clocks are moving with respect to each other, they cannot be precisely coordinated (unless the communication is instantaneous)." The parenthetical reservation is unphysical, and should be left out.

 Suggestion: "However, if the clocks are moving with respect to each other, perfect synchronization would require communication to be instantaneous." (I realize that the original text said "coordinated", not "synchronized", and I appreciate the difference. Feel free to use "perfect coordination", if that is a better fit) ACCEPTED. 

Also: another case of badly delineated references (to "A brief history of timekeeping")

**Annex A Conformance tests**
Missing!

**Annex B Glossary**
It is unclear what entities belong here, vs. chapter 4 (Terms, definitions and abbreviated terms) Also, the problem with the typesetting of "ADMITTED" reoccurs here

**Comment by busstoptaktik** [comment]
There are still a number of suggestions neither ACCEPTED nor REJECTED. Still working on these?

**Comment by chris-little** [comment]
@busstoptaktik Yes - the Temporal DOmain WG meets in 30 seconds!

**Comment by cmheazel** [comment]
It’s not on the portal,  What is the GTM link?

Chuck

**Comment by chris-little** [comment]
1. Please join my meeting, Wed Dec 13th https://www4.gotomeeting.com/join/954119981

**Comment by chris-little** [comment]
@busstoptaktik Are you happy with our responses? We have not yet created a new pdf or html, as there seems to be some infrastructure changes happening, but start reading at [latest version](https://github.com/opengeospatial/Temporal-Abstract-Spec/blob/chris-little-PC1.6/23-049/23-049.adoc). And more comments may arrive until 28 Dec 2023, so we hope to finish processing them before the next meeting on 10 Jan 2024.

**Comment by busstoptaktik** [comment]
> @busstoptaktik Are you happy with our responses

@chris-little I'm mostly happy, but absolutely not happy with the rejection of my suggestion of removing the "(though often not)" part of the **Accountancy** paragraph, so let me make my point a bit more clearly:

The text, as it stands, claims that accountants' use of weeks as time units are "often not" convenient. Let me state straight out that this claim is both inappropriate and unsubstantiated.

**Inappropriate** because it expresses opinion about what is convenient or not in an application domain, where usefulness/conveniency should be entirely up to the practitioners of that domain to decide.

**Unsubstantiated,** because it does not include any evidence for the postulate of inconvenience.

At the very least, the parenthesis should be extended with references to a substantial number of academic papers from business school researchers, clearly and irrefutably concluding that for a **substantial** number of use-cases for weeks/months/quarters as time units, the users would really have been better off using Planck time, nanofortnights, centiyears, or whichever other unit was proposed in the given study.

In other words: You need citations here, and a number of citations compatible with the broad claim of "**often** not".


**Regarding the statement that "Though obervables are measured!",** it is a matter of *what* you measure. You cannot measure the speed of light in vacuum, since that is given by definition. What you actually do, when setting up the classical rotating mirror experiment, by *observing* the speed of light, you *measure* the length of the metre.

**Finally an additional comment:**

The paragraph "If L occurs before M and M occurs before N, that L occurs before N can be correctly deduced" sounds odd to me. It probably should read "If L occurs before M, and M occurs before N, it can be correctly deduced, that L occurs before N"

**Comment by chris-little** [comment]
Texts cahnged to "it can be correctly deduced, that L occurs before N", etc.

**Comment by chris-little** [comment]
To be closed when PR#68 merged.
